### PR TITLE
feat(#15871): Alert when prometheus storage is over 90%

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/giantswarm/issues/15871
Towards https://github.com/giantswarm/giantswarm/issues/16537

## Checklist

I have:

- [x] Described why this change is being introduced
- [x] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

TODO: 
Clean up PVC silences
- [ ] https://github.com/giantswarm/silences/pull/134/files

### CODE REVIEW

File: helm/prometheus-meta-operator/templates/prometheus-rules/disk.management-cluster.rules.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ spec:
       annotations:
         description: '{{`Persistent volume {{ $labels.mountpoint}} on {{ $labels.instance }} does not have enough free space.`}}'
         opsrecipe: low-disk-space/#persistent-volume
-      expr: 100 * node_filesystem_free_bytes{mountpoint=~"(/rootfs)?/var/lib/kubelet.*"} / node_filesystem_size_bytes{mountpoint=~"(/rootfs)?/var/lib/kubelet.*"} < 10
+      expr: 100 * node_filesystem_free_bytes{mountpoint=~"(/rootfs)?/var/lib/kubelet.*",mountpoint!=".*prometheus.*"} / node_filesystem_size_bytes{mountpoint=~"(/rootfs)?/var/lib/kubelet.*",mountpoint!=".*prometheus.*"} < 10
```
Comment (kopiczko) [inline helm/prometheus-meta-operator/templates/prometheus-rules/disk.management-cluster.rules.yml:64]: TIL you can repeat the same label in expression.

**Review by kopiczko — APPROVED** [review]

File: helm/prometheus-meta-operator/templates/prometheus-rules/disk.management-cluster.rules.yml [hunk 2]
```diff
@@ -81,3 +81,15 @@ spec:
         severity: page
         team: biscuit
         topic: storage
+    - alert: PrometheusPersistentVolumeSpaceSpaceTooLow
```
Comment (TheoBrigitte) [inline helm/prometheus-meta-operator/templates/prometheus-rules/disk.management-cluster.rules.yml:84]:
```suggestion
    - alert: PrometheusPersistentVolumeSpaceTooLow
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add support to calculate maximum CPU.
+- Add `PrometheusPersistentVolumeSpaceSpaceTooLow` alert for prometheus storage going over 90 percent.
+
+### Fixed
+
+- Changed prometheus volume space alert ownership to atlas:
+  - `PersistentVolumeSpaceSpaceTooLow` -> `PrometheusPersistentVolumeSpaceSpaceTooLow`
```
Comment (TheoBrigitte) [inline CHANGELOG.md:18]:
```suggestion
  - `PersistentVolumeSpaceTooLow` -> `PrometheusPersistentVolumeSpaceTooLow`
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,6 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add support to calculate maximum CPU.
+- Add `PrometheusPersistentVolumeSpaceSpaceTooLow` alert for prometheus storage going over 90 percent.
```
Comment (TheoBrigitte) [inline CHANGELOG.md:13]:
```suggestion
- Add `PrometheusPersistentVolumeSpaceTooLow` alert for prometheus storage going over 90 percent.
```

**Review by TheoBrigitte — APPROVED** [review]
Beside the typos, this if fine for me.

**Comment by QuentinBisson** [comment]
Thanks @TheoBrigitte

**Pull request merged**